src/components/Fields/TextArea.js

```diff
@@ -29,6 +28,7 @@ const propTypes = {
     onKeyPress: PropTypes.func,
     onKeyUp: PropTypes.func,
     placeholder: PropTypes.string,
+    preventAutoResize: PropTypes.bool,
```

Comment:
I like `preventAutoResize` prop name, but if we want to keep consistency with other bool props I would go for `hasFixedHeight`. Then we can have a matching CSS class too.